pints/_log_likelihoods.py

```diff
@@ -465,6 +465,90 @@ def evaluateS1(self, parameters):
         return L, dL
 
 
+class GaussianIntegratedLogUniformLogLikelihood(pints.ProblemLogLikelihood):
+    r"""
+    Calculates a log-likelihood assuming independent Gaussian-distributed noise
+    at each time point where :math:`p(\sigma)\propto 1/\sigma` has been
+    integrated out of the joint posterior of :math:`p(\theta,\sigma|X)`,
+
+    .. math::
+        \begin{align} p(\theta|X) &= \int_{0}^{\infty} p(\theta, \sigma|X)
+        \mathrm{d}\sigma\\
+        &\propto \int_{0}^{\infty} p(X|\theta, \sigma) p(\theta, \sigma)
+        \mathrm{d}\sigma,\end{align}
+
```

Comment:
This is exact same as next class

Reply:
Well, just do something that makes it clear early on that there are 2 very similar classes, and that this isn't a mistake :D

Reply:
Done.